### PR TITLE
Remove access and references for a member of staff who has left.

### PR DESCRIPTION
**Pull request by sengi**
This removes the user manifest and ssh access to Integration and CI.

### CODE REVIEW

File: modules/users/manifests/petergoddard.pp [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# Creates the petergoddard user
```
Comment (barrucadu) [inline modules/users/manifests/petergoddard.pp]: By deleting this file he'll also need to be removed from staging and production in govuk-secrets, or the next puppet deploy will cause issues

Comment (sengi) [reply modules/users/manifests/petergoddard.pp]: Aha, thanks! I'll hold off on merging this until that's done then.

Comment (sengi) [reply modules/users/manifests/petergoddard.pp]: Confirmed removed from govuk-secrets: https://github.com/alphagov/govuk-secrets/pull/755

**Comment by sengi** [comment]
Superseded by https://github.com/alphagov/govuk-puppet/pull/9403

**Pull request closed without merging**